busypie/condition.py

```diff
@@ -91,6 +96,7 @@ class Condition:
     poll_delay = DEFAULT_POLL_DELAY
     description = None
     return_on_timeout = False
+    at_least = 0
```

Comment:
I wouldn't call it "at_least" as it's the describe the action and not the state

Reply:
we can change wait_time_in_secs to max_wait_time and have min_wait_time as well

Reply:
fixed, due i think it should not be in the scope of this task...